opentech/apply/activity/templates/messages/email/determination.html

```diff
@@ -6,7 +6,15 @@
 {% block content %}
 Your application has been reviewed and the outcome is: {{ determination.clean_outcome }}
 
-{{ determination.message|bleach }}
+{{ determination.message|bleach|striptags }}
 
-Read the complete determination here: {{ request.scheme }}://{{ request.get_host }}{{ determination.get_absolute_url }}
+{% for group in determination.detailed_data.values %}
+{{ group.title }}
+{% for question, answer in group.questions %}
+{% if answer %}{{ question }}:
+{{ answer|bleach|striptags }}{% endif %}
+{% endfor %}
+{% endfor %}
+
+Read the determination here: {{ request.scheme }}://{{ request.get_host }}{{ determination.get_absolute_url }}
```

Comment:
Would it be better for this to be above the text dump above? the text can be quite long.

```
{{ determination.message|bleach|striptags }}

The full determination is included below or you can read the determination here: {{ request.scheme }}://{{ request.get_host }}{{ determination.get_absolute_url }}
```

You can also use: `{% block post_signature_content %}{% endblock %}` to put it after the signature.